setup/setup.sh

```diff
@@ -290,6 +295,8 @@ then generate_data
 fi
 schedule_bq_queries
 
-python -m webbrowser https://datastudio.google.com/datasources/create?connectorId=AKfycbxCOPCqhVOJQlRpOPgJ47dPZNdDu44MXbjsgKw_2-s
+DATASTUDIO_URL=https://datastudio.google.com/datasources/create?connectorId=AKfycbxCOPCqhVOJQlRpOPgJ47dPZNdDu44MXbjsgKw_2-s
+python -m webbrowser ${DATASTUDIO_URL}
```

Comment:
We use python3 other places, does it not matter here?

Reply:
Good catch